src/BuildASub.java

```diff
@@ -5,9 +5,17 @@
     public static void main(String[] args) throws Exception {
         String bread = chooseBread();
         String meat = chooseMeat();
+        String cheese = ChooseCheese();
+        Boolean toast = DoYouWantToasted();
```

Comment:
Naming conventions for booleans fall into the "isX" syntax. So this could have been "isToastRequired"